.github/workflows/test.yml

```diff
@@ -29,6 +29,7 @@ jobs:
 
       - run: python custom_pre_commit/check_doc.py --ignore-files=settings_controller.py,keys_controller.py,featflags_controller.py,degiro_controller.py,oanda_controller.py --ignore-commands=login,al_swe,info_swe,goodness,resources,yf,yolo
       - run: pip install bandit black codespell flake8 mypy pyupgrade safety pylint==2.13.9
+      - run: pip install bandit black codespell flake8 mypy pyupgrade safety pylint
```

Comment:
this addition should be removed. it most likely appeared here because of a conflict